Data/ByteString/Base64/Internal.hs

```diff
@@ -45,8 +45,8 @@ peek8_32 = fmap fromIntegral . peek8
 
 -- | Encode a string into base64 form.  The result will always be a multiple
 -- of 4 bytes in length.
-encodeWith :: EncodeTable -> ByteString -> ByteString
-encodeWith (ET alfaFP encodeTable) (PS sfp soff slen)
+encodeWith :: Bool -> EncodeTable -> ByteString -> ByteString
```

Comment:
Can this parameter be changed to a special-purpose enum ADT instead of a `Bool`?

Reply:
Absolutely it can 😄